csharp/sources/Valkey.Glide/BaseClient.SortedSetCommands.cs

```diff
@@ -24,4 +24,10 @@ public async Task<long> SortedSetAddAsync(ValkeyKey key, SortedSetEntry[] values
 
     public async Task<long> SortedSetAddAsync(ValkeyKey key, SortedSetEntry[] values, SortedSetWhen when = SortedSetWhen.Always, CommandFlags flags = CommandFlags.None)
         => values.Length == 0 ? 0 : await Command(Request.SortedSetAddAsync(key, values, when, flags));
+
+    public async Task<bool> SortedSetRemoveAsync(ValkeyKey key, ValkeyValue member, CommandFlags flags = CommandFlags.None)
+        => await Command(Request.SortedSetRemoveAsync(key, member, flags));
+
+    public async Task<long> SortedSetRemoveAsync(ValkeyKey key, ValkeyValue[] members, CommandFlags flags = CommandFlags.None)
+        => members.Length == 0 ? 0 : await Command(Request.SortedSetRemoveAsync(key, members, flags));
```

Comment:
always send a command
```suggestion
        => await Command(Request.SortedSetRemoveAsync(key, members, flags));
```

+ I'd rename the request to `SortedSetRemove` or `ZRem` (that are non public methods)

Reply:
1. ResponseError: wrong number of arguments for 'zrem' command. Need this for this error.
2. SortedSetRemove. With pattern is present for all commands in the entire file. We will need to refactor all.

Reply:
Updated for 1, updated tests to check for Exception/Errors. Removed the conditional check.